src/stretchy.js

```diff
@@ -166,10 +167,10 @@ const groupLength = function(arg: ParseNode): number {
     }
 };
 
-const svgSpan = function(group: ParseNode, options: Options): domTree.span {
+const svgSpan = function(group: ParseNode, options: Options): DomSpan | SvgSpan {
```

Comment:
The type `SvgSpan` may be confused with the function `svgSpan`.

Reply:
`svgSpan` is actually a constructor for an `SvgSpan` or a list of `SvgSpan`s within a `DomSpan`. So it doesn't feel out of place there. Currently, we have (in increasing order of dependency):

- `SvgSpan` - the fundamental type
- `buildCommon.makeSvgSpan` - `SvgSpan` constructor with `ChildType` in `span<ChildType>` specialized to `svgNode` (aids type inference).
- `stretchy.svgSpan` - to convert a `ParseNode` (that needs SVGs) to a span wrapping one or more SVGs for output.

I'm not sure what to name the last one, but currently, all uses of `stretchy.svgSpan` include the word `stretchy` making the context obvious, so I don't feel as concerned about confusion between the method and the type.

Reply:
Just wanted to check -- are you okay with this? I don't want to merge if you still have any unanswered concerns.

Reply:
@marcianx Yes, I think it's good to go 😄

Reply:
Maybe we should change the factory methods to be something like `createSvgSpan` or `makeSvgSpan`.